src/test/java/space/obminyashka/items_exchange/controller/UserControllerIntegrationTest.java

```diff
@@ -140,6 +140,16 @@ void updateUserPassword_WhenOldPasswordWrong_ShouldThrowInvalidDtoException() th
         assertTrue(mvcResult.getResponse().getContentAsString().contains(getMessageSource("incorrect.password")));
     }
 
+    @Test
+    @WithMockUser(username = "user")
+    void updateUserPassword_WhenUserEnteredOldPassword_ShouldThrowIllegalArgumentException() throws Exception {
+        UserChangePasswordDto userChangePasswordDto = new UserChangePasswordDto(CORRECT_OLD_PASSWORD, CORRECT_OLD_PASSWORD, CORRECT_OLD_PASSWORD);
```

Comment:
Just FYI, you can use **var** instead of the class, which could be `var userChangePasswordDto = new UserChangePasswordDto(...) `.
But it's up to you, of course.